communication/templates/robot.html

```diff
@@ -0,0 +1,38 @@
+{{define "robot-page"}}
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <!-- Primary Meta Tags -->
+    <title>Scribble.rs - Join Lobby</title>
+    <meta name="title" content="Scribble.rs - Join Lobby">
+    <meta name="description" content="{{.MaxPlayers}} Players - {{.MaxRounds}} Rounds">
+
+    <!-- Open Graph / Facebook -->
+    <meta property="og:type" content="website">
+    <meta property="og:title" content="Scribble.rs - Join Lobby">
+    <meta property="og:description" content="{{.MaxPlayers}} Players - {{.MaxRounds}} Rounds">
+    <meta property="og:image" content="/resources/favicon.png">
+
+    <!-- Twitter -->
+    <meta property="twitter:card" content="summary_large_image">
+    <meta property="twitter:title" content="Scribble.rs - Join Lobby">
+    <meta property="twitter:description" content="{{.MaxPlayers}} Players - {{.MaxRounds}} Rounds">
+    <meta property="twitter:image" content="/resources/favicon.png">
+    <link rel="stylesheet" type="text/css" href="/resources/base.css" />
```

Comment:
`@font.face` declaration is missing. Has this been left out intentionally?
If so, we might want to add a comment as to why these aren't here, but for example on the error page.

Reply:
I copied from the old `error.html`. It's missing `{{.RootPath}}` I think. I'll check it.